lib/puppet-lint/plugins/check_parameter_documentation.rb

```diff
@@ -45,8 +45,19 @@ def check
         end
       rescue StopIteration; end
 
+      # warn about duplicates
+      doc_params.select.with_index do |param, idx|
+        idx != doc_params.find_index { |p| param[:value] == p[:value] }
+      end.each do |p|
+        notify :warning, {
+          :message => "Duplicate parameter documentation for #{p[:value]}",
+          :line    => p[:token].line,
+          :column  => p[:token].column
```

Comment:
The column will have a mismatch since there is a regex. You may want to consider `p[:token].column + p[:token].value.match(/\A\s*/)[0].length` (which is OK because all regexes around line 11 start with whitespace.

Reply:
Tokens with `@param` have two sets of whitespace.  eg `p[:token].value == ' @param foo'`